### PR TITLE
feat: Add Access button to user account access component

### LINKED ISSUE #2447: Change create button in Access to Material Floating Action Button
Remove the _Add Access_ button from `gf-access-table` (see _My Ghostfolio_ > _Access_) and add a `mat-fab` button to `gf-user-account-access` instead.
There is a very similar example in _Admin Control_ > _Market Data_.

create-account-access

### PR DESCRIPTION
**Pull request by Dhoni77**
Closes #2447 

- Remove add access button from gf-access-table
- Added access button to gf-user-account-access

### CODE REVIEW

**Comment by Dhoni77** [comment]
cc @dtslvr

File: apps/client/src/app/components/user-account-access/user-account-access.html [hunk 1]
```diff
@@ -10,8 +10,19 @@
...
     [accesses]="accesses"
-    [hasPermissionToCreateAccess]="hasPermissionToCreateAccess"
     [showActions]="hasPermissionToDeleteAccess"
     (accessDeleted)="onDeleteAccess($event)"
   ></gf-access-table>
+  <div *ngIf="hasPermissionToCreateAccess" class="fab-container">
+    <a
+      class="align-items-center d-flex justify-content-center"
+      color="primary"
+      i18n
```
Comment (dtslvr) [inline apps/client/src/app/components/user-account-access/user-account-access.html:20]: As there is no button label, you can remove the i18n directive 🙂

**Review by dtslvr — CHANGES_REQUESTED** [review]
Looks fine! Thanks for your contribution, @Dhoni77 🙂

File: apps/client/src/app/components/user-account-access/user-account-access.html (hunk 1 above)
Comment (Dhoni77) [reply apps/client/src/app/components/user-account-access/user-account-access.html]: done

**Comment by Dhoni77** [comment]
cc @dtslvr

**Pull request merged**